this is standard since python3.8, stop using pypi modules for this
diff --git a/chaospy/__init__.py b/chaospy/__init__.py
index 9430314..f0c5321 100644
--- a/chaospy/__init__.py
+++ b/chaospy/__init__.py
@@ -6,7 +6,7 @@ This module contains tools for performing uncertainty quantification of models.
 """
 import logging
 import os
-import importlib_metadata
+import importlib.metadata
 
 from numpoly import *
 
@@ -29,7 +29,7 @@ from chaospy.regression import *
 from chaospy.external import *
 from chaospy.recurrence import *
 
-__version__ = importlib_metadata.version("chaospy")
+__version__ = importlib.metadata.version("chaospy")
 
 
 def configure_logging():
